src/pages/Home.js

```diff
@@ -1,10 +1,14 @@
-import React from 'react';
+import React, {useContext} from 'react';
 import { Link } from 'react-router-dom';
+import {AuthContext} from "../context/AuthContext";
 
 function Home() {
+
+  const {isAuth} = useContext(AuthContext);
+
   return (
     <>
-      <h1>Homepagina</h1>
+      <h1>Homepagina{isAuth ? "Je bent ingelogd" : "Vul je login gegevens in"}</h1>
```

Comment:
Ik denk dat dit een goed idee is. Hierdoor weten gebruikers duidelijk of hun home page is ingelogd of niet.